pyproject.toml

```diff
@@ -18,7 +18,7 @@ license = {file="LICENSE"}
 dynamic = ["version"]
 
 # Dependencies
-requires-python = ">=3.11"
+requires-python = ">=3.9"
```

Comment:
This is the same as benchcab, resolves an installation issue. Does not have any issues on the hpcpy side of things.